Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:alpine
+
+RUN mkdir -p /usr/src/app
+
+RUN apk add --no-cache make bash
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+LABEL maintainer="cncf-falco-dev@lists.cncf.io"
+
+RUN make
+
+CMD ["./event-generator", "run", "syscall.Write"]
```

Comment:
I would not use any arguments here. IMHO, it's OK that the container does nothing by default (it will just print the help message). Users will have to pass the arguments depending on their use case.